### PR TITLE
Bug fixes

### PR DESCRIPTION
**Pull request by jennacryan**

### CODE REVIEW

File: db/migrate/20151012010940_create_users.rb [hunk 1]
```diff
@@ -3,7 +3,7 @@ def change
     create_table :users do |t|
       t.string :name
       t.string :email
-      t.string :password_digest
+      t.string :password
```
Comment (michaelRadbel) [inline db/migrate/20151012010940_create_users.rb]: In general editing existing migrations is not a good idea, typically its best to create a new migration

Why did you change this to password? password_digest is used by the bcrypt gem and is straight from the Github example on its usage.

**Comment by michaelRadbel** [comment]
Would take out .idea files, everything else looks good :+1:

File: .idea/workspace.xml [hunk 2]
```diff
@@ -0,0 +1,849 @@
...
+    <entry file="file://$PROJECT_DIR$/app/models/user.rb">
+      <provider selected="true" editor-type-id="text-editor">
+        <state vertical-scroll-proportion="0.19783616">
+          <caret line="8" column="0" selection-start-line="8" selection-start-column="0" selection-end-line="8" selection-end-column="0" />
+          <folding />
+        </state>
+      </provider>
+    </entry>
+  </component>
+</project>
```
Comment (camsb) [inline .idea/workspace.xml]: Get rid of this file. All of these are generated by RubyMine. Also, make sure you add the default new line into your edits, or Github will always complain. If you use Sublime, get into your settings ("Preferences.sublime-settings") and add this line "ensure_newline_at_eof_on_save": true,

File: .idea/vcs.xml [hunk 3]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="VcsDirectoryMappings">
+    <mapping directory="$PROJECT_DIR$" vcs="Git" />
+  </component>
+</project>
```
Comment (camsb) [inline .idea/vcs.xml]: Get rid of this file

File: .idea/modules.xml [hunk 4]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="ProjectModuleManager">
+    <modules>
+      <module fileurl="file://$PROJECT_DIR$/.idea/foodfinder.iml" filepath="$PROJECT_DIR$/.idea/foodfinder.iml" />
+    </modules>
+  </component>
+</project>
```
Comment (camsb) [inline .idea/modules.xml]: Get rid of this file

File: .idea/misc.xml [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+    <OptionsSetting value="true" id="Remove" />
+    <OptionsSetting value="true" id="Checkout" />
+    <OptionsSetting value="true" id="Update" />
+    <OptionsSetting value="true" id="Status" />
+    <OptionsSetting value="true" id="Edit" />
+    <ConfirmationsSetting value="0" id="Add" />
+    <ConfirmationsSetting value="0" id="Remove" />
+  </component>
+  <component name="ProjectRootManager" version="2" project-jdk-name="RVM: ruby-2.2.1" project-jdk-type="RUBY_SDK" />
+</project>
```
Comment (camsb) [inline .idea/misc.xml]: Get rid of this file

File: .idea/foodfinder.iml [hunk 6]
```diff
@@ -0,0 +1,197 @@
...
+    <orderEntry type="library" scope="PROVIDED" name="turbolinks (v2.5.3, RVM: ruby-2.2.1) [gem]" level="application" />
+    <orderEntry type="library" scope="PROVIDED" name="tzinfo (v1.2.2, RVM: ruby-2.2.1) [gem]" level="application" />
+    <orderEntry type="library" scope="PROVIDED" name="uglifier (v2.7.2, RVM: ruby-2.2.1) [gem]" level="application" />
+    <orderEntry type="library" scope="PROVIDED" name="web-console (v2.2.1, RVM: ruby-2.2.1) [gem]" level="application" />
+  </component>
+  <component name="RModuleSettingsStorage">
+    <LOAD_PATH number="0" />
+    <I18N_FOLDERS number="1" string0="$MODULE_DIR$/config/locales" />
+  </component>
+</module>
```
Comment (camsb) [inline .idea/foodfinder.iml]: Get rid of this file

File: .idea/.name [hunk 7]
```diff
@@ -0,0 +1 @@
+foodfinder
```
Comment (camsb) [inline .idea/.name]: Get rid of this file

File: .idea/.generators [hunk 8]
```diff
@@ -0,0 +1,8 @@
...
...eratorsGroup><Generator name="active_record:migration" /><Generator name="active_record:model" /><Generator name="active_record:observer" /><Generator name="active_record:session_migration" /><Generator name="controller" /><Generator name="erb:controller" /><Generator name="erb:mailer" /><Generator name="erb:scaffold" /><Generator name="generator" /><Generator name="helper" /><Generator name="integration_test" /><Generator name="mailer" /><Generator name="metal" /><Generator name="migration" /><Generator name="model" /><Generator name="model_subclass" /><Generator name="observer" /><Generator name="performance_test" /><Generator name="plugin" /><Generator name="resource" /><Generator name="scaffold" /><Generator name="scaffold_controller" /><Generator name="session_migration" /><Generator name="stylesheets" /><Generator name="test_unit:controller" /><Generator name="test_unit:helper" /><Generator name="test_unit:integration" /><Generator name="test_unit:mailer" /><Generator name="test_unit:model" /><Generator name="test_unit:observer" /><Generator name="test_unit:performance" /><Generator name="test_unit:plugin" /><Generator name="test_unit:scaffold" /></GeneratorsGroup></Settings>
```
Comment (camsb) [inline .idea/.generators]: Get rid of this file

**Comment by bboe** [comment]
I was looking over the PR comments and I have a little feedback.

@camsb do you think you could convey the same meaning by saying "Could you please remove this file?", or "Is this file necessary?" rather than "Get rid of this file"?

In general I find people to be more receptive to questions in pull requests than commands. The following is a great talk I attended this year which I think describes the purpose better than I can (I linked to the relevant section):
https://youtu.be/PJjmw9TRB7s?t=678

**Comment by camsb** [comment]
Yea of course! I mentioned in some other comment why all the .ini files shouldn't be included and what they were. I was trying to mark which ones I meant, but you are right nonetheless.

Sent from my iPhone

On Nov 3, 2015, at 12:58 AM, Bryce Boe <notifications@github.com<mailto:notifications@github.com>> wrote:

I was looking over the PR comments and I have a little feedback.

@camsbhttps://github.com/camsb do you think you could convey the same meaning by saying "Could you please remove this file?", or "Is this file necessary?" rather than "Get rid of this file"?

In general I find people to be more receptive to questions in pull requests than commands. The following is a great talk I attended this year which I think describes the purpose better than I can (I linked to the relevant section):
https://youtu.be/PJjmw9TRB7s?t=678

##
